CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+
+## 🤓 Initial set up
+
+### 🪟 Note for Windows users
+
+Windows Subsystem for Linux can be a much more versatile and familiar environment for software development on Windows. Everything from installing `git` and other dependencies to using command line tools that will be familiar to the wider community of software developers are more likely to be easier under WSL. While some parts of some Openverse projects may be able to be developed under native Windows, you will have a much smoother time with WSL as our command runners (`just` and `pnpm`) assume a unix-type environment (Linux and macOS). We would like to support native Windows development, but we do not currently have any active contributors who use Windows on a daily basis and do not have the time to spend supporting it. Likewise, many other OSS projects will not be configured to allow easy local development under Windows. WSL can help there as well.
+
+Installation instructions for WSL on Windows 10 and 11 can be found [here on Microsoft's official documentation website](https://docs.microsoft.com/en-us/windows/wsl/install).
+
+Of course, if you would like to contribute to Openverse by making our projects work natively on Windows, PRs for this are always welcome.
+
+### ✍️ General set up
+
+Skip any steps you've already completed on your own. This is meant as an exhaustive list for brand-new devs who might not have these tools yet.
+
+1. Install `git`
+    * Most Linux distributions and macOS will already have this installed. Open up your terminal app and type `which git`. If the response is anything other than `git not found` then you've already got it installed.
+    * If you need to install it, follow the official instructions here: https://git-scm.com/downloads
+2. Install a text editor.
+    * [PyCascades, an event we've attended previously, has a great list of common editors](https://mentored-sprint-for-diverse-beginners.readthedocs.io/01_contributor_resources.html#text-editors). VSCode in particular is very popular for Python and JavaScript developers alike.
+3. Create a GitHub account <https://github.com/signup>
+4. Install pyenv or Anaconda
+    * These tools make it simpler to manage multiple different versions of Python on a single machine. This is useful for contributing to multiple projects, each of which may be using a different specific version of Python.
+    * Skip this if you only want to do a JavaScript issue
+    * https://github.com/pyenv/pyenv
+    * https://www.anaconda.com/products/individual
+5. Install volta
+    * Similar to pyenv/Anaconda but for Node.js
+    * Skip this if you only want to do a Python issue
+    * https://volta.sh/
+6. Install Docker
+    * Skip this if you only want to do a JavaScript issue
+    * Windows with WSL: https://dev.to/felipecrs/simply-run-docker-on-wsl2-3o8
```

Comment:
```suggestion
    * Windows with WSL: https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers#install-docker-desktop
```

This looks simpler, and will probably be updated more often since it's from Microsoft.

Reply:
I think it is slightly different than the blog post linked as it still encourages you to run other things outside of WSL (which means our scripts and stuff still won't work). It seems like a better resource long term though. Generally I don't want to get too bogged down in supporting Windows at the moment without any active or potential contributors relying on Windows support.